frontend/src/components/PaymentGateway/PaymentGateway.tsx

```diff
@@ -16,8 +16,9 @@ const fetchApiKey = async () => {
 export const SubscriptionPlans: React.FC<{ stripePromise: Promise<Stripe | null> }> = ({ stripePromise }) => {
     const { user, organization } = useAppContext();
 
-    const [plans, setPlans] = useState<any[]>([]);
-    const currentPlan = organization?.subscriptionId ? 1 : 0;
+    const [currentPlan, setCurrentPlan] = useState(organization.subscriptionId ? 1 : 0);
```

Comment:
we may have more than two plans in the future, consider this 

`enum PlanStatus {
    NoSubscription = 0,
    Subscribed = 1
}

const [currentPlan, setCurrentPlan] = useState<PlanStatus>(organization.subscriptionId ? PlanStatus.Subscribed : PlanStatus.NoSubscription);
`

Reply:
@ramirezmorac2 take a look of that

Reply:
@namtran6701 I remove that code is no longer need the plans we can getting from the product prices on stripe

Reply:
Understood!